Clappr-tvOS/Classes/Base/Options.swift

```diff
@@ -0,0 +1,20 @@
+public typealias Options = [String: Any]
```

Comment:
You can extract the options of tvOS to shared options, what you think?

```swift
#if os(tvOS)
public let kAutoPlay = "autoplay"
public let kLoop = "loop"
public let kMetaData = "metadata"
public let kMetaDataTitle = "mdTitle"
public let kMetaDataDescription = "mdDescription"
public let kMetaDataDate = "mdDate"
public let kMetaDataArtwork = "mdArtwork"
#endif
```

Reply:
Hey @CristianDRM I'm going to change this in the future along with the other changes, ok?